base/linalg/symmetric.jl

```diff
@@ -423,10 +470,8 @@ function expm{T}(A::Hermitian{T})
 end
 
 for (funm, func) in ([:logm,:log], [:sqrtm,:sqrt])
-
     @eval begin
-
-        function ($funm)(A::Symmetric)
+        function ($funm){T<:Real}(A::Symmetric{T})
```

Comment:
Cool. Useful to write that down somewhere, I wouldn't have remembered that detail from a linear algebra textbook.